.github/ISSUE_TEMPLATE/ui_story.md

```diff
@@ -0,0 +1,42 @@
+---
+name: UI Story
+about: Explain the User story, regarding UI related changes.
+title: 'UI Story'
+labels: ui-story, user-story
+assignees: ''
+
+---
+
+<!-- /!\ Please ensure that you are NOT disclosing any customer information without their consent /!\ -->
+
+## User Story
+As an Automate User,
+I want to do this on the Automate UI.
+
+## Acceptance Criteria
+- This page should have these changes.
+
+## Implementation Details
+- In this page, add this new button.
+- On click of this button, trigger this function.
+
+## UI Design
+New Design Preview Image:
+![Screenshot 2021-08-05 at 6.25.24 PM.png](https://images.zenhubusercontent.com/6008075cc8db73075cc33e45/2da608f3-6ef8-4b87-927a-56df455729dc)
+
+CSS and Other settings can be found [here](http://ui-figma-design-preview.com)
+
+## Definition of Done
+
+- All things specified in User Story Acceptance Criteria should be fulfilled.
+- UI should exactly match UX design.
+- Cypress Coverage for new Feature.
+- All Exceptions are Handled Properly
+- Ensure logs have no unnecessary data.
+- Test coverage for new feature is done to at-least 70%.
+- Docs changes PR is Raised.
+- Swagger Documentation updated.
+- Smoke Test done.
+- Ensure Build and Integration Pipelines are Green.
+- PR has 2 approvers.
+- All Code Review Comments are Resolved.
```

Comment:
Also valuable can be:
	Screen shot of the feature developed is provided in the issue.
         Ensure compatibility with all supporting browser
         Demoed the feature to UX
         Link to the UX wireframe/Requirement Document

Reply:
done